catalog_validation/validation.py

```diff
@@ -15,7 +15,7 @@ def validate_catalog(catalog_path):
 
     for file_dir in os.listdir(catalog_path):
         complete_path = os.path.join(catalog_path, file_dir)
-        if file_dir.startswith('.') or not os.path.isdir(complete_path) or file_dir == 'library':
+        if file_dir.startswith('.') or not os.path.isdir(complete_path) or file_dir == 'library' or file_dir == 'docs':
```

Comment:
Let's have `file_dir in ('library', 'docs')` please

Reply:
Did as you asked, but python is not my strongest language tbh.